src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
+    private String type;
+
+    public Figure(String color, String type) {
+        this.color = color;
+        this.type = type;
+    }
+
+    @Override
```

Comment:
The `draw` method implementation should not begin with an empty line. Remove the redundant empty line before the method signature.